scraper/doctolib/doctolib.py

```diff
@@ -200,6 +223,13 @@ def get_appointments(self, request: ScraperRequest, start_date: str, visit_motiv
                 if not first_availability or slot_list[0] < first_availability:
                     first_availability = slot_list[0]
                     motive_availability = True
+
+            for interval in INTERVAL_SPLIT_DAYS:
+                if start_date <= date_plus_day(start_date_original, interval):
+                    if availability.get('date') is not None:
+                        if availability.get('date') <= date_plus_day(start_date_original,interval):
+                            count_next_appointments[f"{interval}_days"] += len(availability.get('slots', None))
```

Comment:
`len()` n'existe pas pour `None` (ça lève une exception), donc si on veut gérer le cas où `slots` est absent, ce serait mieux de faire comme suit :
```suggestion
                            count_next_appointments[f"{interval}_days"] += len(availability.get('slots', []))
```